airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,73 @@
+from typing import Dict, Callable, Any
+
+from airflow.exceptions import AirflowException
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks.clickhouse_hook import ClickHouseHook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+        self,
+        sql: str = None,
+        clickhouse_conn_id: str = 'clickhouse_default',
```

Comment:
Let's make `ClickHouseHook._DEFAULT_CONN_ID` a public const defined in `airflow_clickhouse_plugin.__init__`. And use it as a protected field both of `ClickHouseHook._DEFAULT_CONN_ID` and `ClickHouseSqlSensor._DEFAULT_CONN_ID`. And use it in the `__init__` method here as follows:
https://github.com/whisklabs/airflow-clickhouse-plugin/blob/a88b4249a0db0b06866d5a4bedc2e08254773af1/airflow_clickhouse_plugin/hooks/clickhouse_hook.py#L8-L13